src/carplay.cs

```diff
@@ -1008,4 +1289,556 @@ interface CPTemplateApplicationDashboardScene {
+
+		[Internal]
+		[Export ("initWithFullName:phoneOrEmailAddress:leadingConfiguration:trailingConfiguration:detailText:trailingText:")]
+		IntPtr InitWithFullName (string fullName, string phoneOrEmailAddress, CPMessageListItemLeadingConfiguration leadingConfiguration, [NullAllowed] CPMessageListItemTrailingConfiguration trailingConfiguration, [NullAllowed] string detailText, [NullAllowed] string trailingText);
+
+		[NullAllowed, Export ("conversationIdentifier")]
+		string ConversationIdentifier { get; set; }
+
+		[NullAllowed, Export ("phoneOrEmailAddress")]
+		string PhoneOrEmailAddress { get; set; }
+
+		[Export ("leadingConfiguration", ArgumentSemantic.Strong)]
+		CPMessageListItemLeadingConfiguration LeadingConfiguration { get; set; }
+
+		[NullAllowed, Export ("trailingConfiguration", ArgumentSemantic.Strong)]
+		CPMessageListItemTrailingConfiguration TrailingConfiguration { get; set; }
+
+		[NullAllowed, Export ("detailText")]
+		string DetailText { get; set; }
+
+		[NullAllowed, Export ("trailingText")]
+		string TrailingText { get; set; }
+
+		[Field ("CPMaximumMessageItemImageSize")]
+		CGSize MaximumMessageItemImageSize { get; }
+
+		[NullAllowed, Export ("text")]
+		string Text { get; set; }
+
+		[NullAllowed, Export ("userInfo", ArgumentSemantic.Strong)]
+		NSObject UserInfo { get; set; }
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (NSObject))]
+	interface CPMessageListItemLeadingConfiguration
+	{
+		[Export ("unread")]
+		bool Unread { [Bind ("isUnread")] get; }
+
+		[Export ("leadingItem")]
+		CPMessageLeadingItem LeadingItem { get; }
+
+		[NullAllowed, Export ("leadingImage")]
+		UIImage LeadingImage { get; }
+
+		[Export ("initWithLeadingItem:leadingImage:unread:")]
+		IntPtr Constructor (CPMessageLeadingItem leadingItem, [NullAllowed] UIImage leadingImage, bool unread);
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (NSObject))]
+	interface CPMessageListItemTrailingConfiguration
+	{
+		[Export ("trailingItem")]
+		CPMessageTrailingItem TrailingItem { get; }
+
+		[NullAllowed, Export ("trailingImage")]
+		UIImage TrailingImage { get; }
+
+		[Export ("initWithTrailingItem:trailingImage:")]
+		IntPtr Constructor (CPMessageTrailingItem trailingItem, [NullAllowed] UIImage trailingImage);
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (14,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface CPNowPlayingButton : NSSecureCoding
+	{
+		[Export ("initWithHandler:")]
+		IntPtr Constructor ([NullAllowed] Action<CPNowPlayingButton> handler);
+
+		[Export ("enabled")]
+		bool Enabled { [Bind ("isEnabled")] get; set; }
+
+		[Export ("selected")]
+		bool Selected { [Bind ("isSelected")] get; set; }
+
+		[Field ("CPNowPlayingButtonMaximumImageSize")]
+		CGSize MaximumImageSize { get; }
+	}
+
+	[BaseType (typeof (CPNowPlayingButton))]
+	interface CPNowPlayingImageButton
```

Comment:
missing availability attributes (for us that implies it was available since iOS 7)

Reply:
`[DisableDefaultCtor]`